lib/rides/models/ride.dart

```diff
@@ -14,6 +15,7 @@ class Ride extends Model {
 
   final int driveId;
   final int riderId;
+  final Profile? rider;
```

Comment:
if we have the rider for a ride, do we really need the riderId additionally?
and we could also add the driver but thats maybe in another PR

Reply:
The problem ist that the `rider` is only present if you do the join in the SQL query. But `rider_id` is always present, and needed for the Json export.

Yes of course adding `driver` will be a logical step when we need it.